platform/install/configure/config.mdx

```diff
@@ -93,6 +93,30 @@ config:
       X-XSS-Protection: 1; mode=block
 ```
 
+### Setting additional certificate authority for Platform
+If you decide to configure vCluster Platform helm release with  `additionalCA`, it will be automatically used by all vClusters managed by the vCluster Platform to establish a connection between vClusters and the Platform. You don't need to tweak individual vCluster configuration to enable this.
+This will enable secure communication between your vClusters and the vCluster Platform.
+```yaml
+additionalCA: "(base64 encoded CA)"
+```
+
+Also, if you need to import externally managed vCluster to the vCluster Platform, you can also enable a secure connection by grabbing the value from the  existing vCluster Platform helm release, you can do this by running:
+
+```shell
+helm get values --namespace [namespace where vCluster Platform is deployed] [loft helm release name] --all | grep additionalCA
+```
+
+Then, using you can import vCluster using vCluster CLI command:
+```shell
+  vcluster platform add vcluster [your vCluster name] \
```

Comment:
Typically when we do commands that require replace statements we have a style that is like

`<VCLUSTER_NAME>`
`<VCLUSTER_NAMESPACE>`
`<PROJECT_NAME>`

And then after the command it gives a hint that you should replace those values with a description of what they mean. 

Example of what I mean is shown here: https://www.vcluster.com/docs/vcluster/manage/upgrade-version?x0=1#upgrade-vcluster-version

Reply:
:+1: changed